src/triage/component/catwalk/model_trainers.py

```diff
@@ -154,7 +154,7 @@ def _save_feature_importances(self, model_id, feature_importances, feature_names
                 model_id=model_id,
                 feature_importance=0,
                 feature='Algorithm does not support a standard way' +
-                        'to calculate feature importance.',
+                        ' to calculate feature importance.',
```

Comment:
We should probably define this as a constant in one of the files and import it from the other. It would be way too easy for somebody to come in and reformat this line or add a word and break the other check.